app/_landing_pages/insomnia/documents.yaml

```diff
@@ -0,0 +1,239 @@
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Send requests in Insomnia
+            cta:
+              text: Learn more
+              url: /
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Chain requests in Insomnia
+            cta:
+              text: Learn more
+              url: /how-to/chain-requests/
+              align: end
+  
+  - header:
+      type: h2
+      text: "Test API specs"
+    columns:
+      - blocks:
+          - type: image
+            config:
+              url: /assets/images/insomnia/document-test.png
+              alt_text: Image of document tests in a test suite.
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: | 
+                    You can create tests for the requests populated from your specification in the Tests tab of a design document. You can use Insomnia's pre-configured test examples by editing a test and pressing **Control + Space**.
+                    
+                    You can manually run an individual test or an entire test suite from Insomnia. You can also automate tests with the Inso CLI.
+
+                    If you want to run pre-request or after-response tests, you can do that from the **Collection** tab under **Scripts**.
+          - type: button
+            config:
+              align: left
+              text: Learn more about testing APIs
+              url: /insomnia/test/
+
+  - header:
+      type: h3
+      text: "Get started"
+    columns:
+      - blocks:
+        - type: card
+          config:
+            title: Automate tests in Insomnia
+            cta:
+              text: Learn more
+              url: /how-to/automate-tests/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Write tests for HTTP status codes
+            cta:
+              text: Learn more
+              url: /how-to/write-http-status-tests/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Write tests for content types
+            cta:
+              text: Learn more
+              url: /how-to/write-content-type-tests/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Write tests for headers in the response
+            cta:
+              text: Learn more
+              url: /how-to/write-headers-in-response-tests/
+              align: end
+  - header:
+      text: "Frequently Asked Questions"
+      type: h2
+    columns:
+      - blocks:
+        - type: faqs
+          config:
+            - q: What's the difference between a collection and a design document?
+              a: You can't create, lint, and edit a spec from a collection. If you want to use version control sync, you should use a collection instead.
```

Comment:
I don't understand this:
> If you want to use version control sync, you should use a collection instead.

Because you can sync a design document with Git.

Reply:
To me the difference is that a design document houses three interconnected pieces together. So like the design document is an object that can contain a spec, a collection, and tests. Then you can interact with those pieces using the features of Insomnia. 

A collection feels like it is just a collection of endpoints that can either be generated, imported, and just manually pasted in. 

I don't think there are any differences between the collection, and the collection within the design document as far as capabilities are concerned.